ztp/resource-generator/Containerfile

```diff
@@ -15,17 +15,27 @@ WORKDIR $PGT_ROOT
 RUN make build
 WORKDIR $SC_ROOT
 RUN make build
+WORKDIR /go/src/
+RUN curl -L $(curl https://api.github.com/repos/stolostron/policy-generator-plugin/releases/latest | jq -r '.tarball_url') -o policy-generator-plugin.tar.gz
```

Comment:
Running Curl as part of 'podman build' won't work for downstream builds; it needs to be completely offline-capable.

Reply:
is there a way we can validate the downstream build ?